ow_faults_injection/cfg/Faults.cfg

```diff
@@ -100,7 +100,7 @@ ft_sensor_faults.add("signal_noise_failure", double_t, 0,
 
 power_faults = gen.add_group("power_faults")
 power_faults.add("high_power_draw",   double_t, 0,
-    "High power draw increment", 1.0, 1.0, 20.0)
+    "High power draw increment", 1.0, 1.0, 400.0)
```

Comment:
I'm guessing that this value should be 20 times the number of cells.  So for 24 cells, shouldn't it be 480?  And we'd also need to document that cell count and this max needs to be kept in sync.

Reply:
The exact value is probably something we should briefly discuss with Chetan, though yes, 20 times the number of cells is probably a good benchmark. I just set a basic higher value for general testing purposes.

That said, if we wanted the max wattage per cell to be 20 then we'd probably also want to take into account the 1W baseline draw each cell has, so the actual max would be something like 456W instead. Such a value seems kind of arbitrary? Again, it's something to discuss.

Reply:
I don't think this needs to be done for this PR, but I think it makes the most sense to make this value 20, and track it per cell, ideally issuing only a single warning when the value is breached for any cell instead of 24 warnings for each cell.

Reply:
For what it's worth, I updated the mechanism for that specific warning. It'll now only output one warning message that lists every node that went over the 20W cap.

That said, because power is distributed evenly across all nodes, if one node is over the cap then every other node is also over the cap, so all nodes are listed any time the limit is exceeded. It also might not make sense to individually list each node in such a warning anyways; the point is that the pack acts as a single larger battery, right?

Reply:
I agree with the previous comment, that warning seems sensible if the battery as a whole exceeds a power threshold.

The warning may make sense for individual cells if somehow one can hit the threshold without the whole battery reaching its threshold.   This doesn't seem possible with our current battery,  but maybe it would become possible if we start modeling intra-battery faults and failures.  

Would be good for @chetankul to review these comments.